etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/EtorDomainRegistration.java

```diff
@@ -131,4 +138,16 @@ DomainResponse handleOrders(DomainRequest request) {
         OrderResponse orderResponse = new OrderResponse(orders);
         return domainResponseHelper.constructOkResponse(orderResponse);
     }
+
+    DomainResponse handleMetadata(DomainRequest request) {
+        try {
+            String uniqueId = request.getBody();
```

Comment:
I think it makes more sense to get the unique ID from the URL path.  E.g. `/v1/etor/metadata/{uniqueId}`.

Reply:
Agreed. I'll change that

Reply:
Done